homeassistant/components/aws/const.py

```diff
@@ -10,8 +13,18 @@
 CONF_CREDENTIAL_NAME = "credential_name"
 CONF_CREDENTIALS = "credentials"
 CONF_NOTIFY = "notify"
+CONF_IMAGE_PROCESSING = "image_processing"
 CONF_PROFILE_NAME = "profile_name"
 CONF_REGION = "region_name"
 CONF_SECRET_ACCESS_KEY = "aws_secret_access_key"
 CONF_SERVICE = "service"
 CONF_VALIDATE = "validate"
+
+CONF_COLLECTION_ID = "collection_id"
+CONF_IDENTIFY_FACES = "identify_faces"
+CONF_DETECTION_ATTRIBUTES = "detection_attributes"
+CONF_SAVE_FILE_FOLDER = "save_file_folder"
+CONF_SAVE_FILE_TIMESTAMP = "save_file_timestamp"
+
+EVENT_OBJECT_DETECTED = "image_processing.detect_object"
```

Comment:
Would this not be `"rekognition.object_detected"`?

Reply:
Agree - was trying to make it more generic but I think it would just cause for confusion for the user, ill rename 😄

Reply:
I recall there was guidance in the dev docs on this point but cannot locate it